docs improvement (#1765)
spidernet-io/spiderpool: docs/usage/spider-subnet.md

3. For annotation `ipam.spidernet.io/ippool-ip-number`, you can use '2' for fixed IP number or '+2' for flexible mode.
   The value '+2' means the SpiderSubnet auto-created IPPool will add 2 more IPs based on your application replicas.
   If you choose to use flexible mode, the auto-created IPPool IPs will expand or shrink dynamically by your application replicas.

>    This is an optional annotation. If you do not set it, it will default to use ConfigMap `spiderpool-conf` property `clusterSubnetDefaultFlexibleIPNumber` as the flexible mode flexible IP number. Refer to [config](../concepts/config.md) for details.

Reviewer: ```suggestion
   This is an optional annotation. If left unset, it will use the `clusterSubnetDefaultFlexibleIPNumber` property from the `spiderpool-conf` ConfigMap as the flexible IP number in flexible mode.
   Refer to [config](../concepts/config.md) for more information.
```
